applications/FemToDemApplication/custom_processes/update_dem_kinematics_process.cpp

```diff
@@ -51,6 +51,7 @@ void UpdateDemKinematicsProcess::UpdateKinematics(
 {
     const array_1d<double,3> coordinates = rNode->Coordinates();
```

Comment:
Add references if needed to avoid copies

Reply:
I was tempted to do that but I noticed that in some cases is kind of dangerous when crossing refs...